x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/recent_activity.scss

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+.activity {
+  display: flex;
+  justify-content: space-between;
+  padding: $euiSizeM;
+  font-size: $euiFontSizeS;
+
+  &--error {
+    font-weight: $euiFontWeightSemiBold;
+    color: $euiColorDanger;
+    background: rgba($euiColorDanger, 0.1);
+
+    &__label {
+      margin-left: $euiSizeS * 1.75;
+      font-weight: $euiFontWeightRegular;
+      text-decoration: underline;
+      opacity: 0.7;
+    }
+  }
+
+  &__message {
+    flex-grow: 1;
+  }
+
+  &__date {
+    flex-grow: 0;
+  }
+
+  & + & {
+    border-top: $euiBorderThin;
+
+    &--error {
+      border-color: transparent;
+    }
```

Comment:
All this is lookin' real good. To fix that error border problem (_error border error_?) we can just remove these lines.
```suggestion
```